src/manifest.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2023 Design Barn Inc.
+ */
+
+export enum PlayMode {
+  Bounce = 'bounce',
+  Normal = 'normal',
+}
+
+export interface ManifestAnimation {
+  autoplay?: boolean;
+
+  // Define playback direction 1 forward, -1 backward
+  direction?: number;
+
+  id: string;
+
+  // If loop is a number, it defines the number of times the animation will loop
+  loop?: boolean;
+
+  // Choice between 'bounce' and 'normal'
+  playMode?: PlayMode;
+
+  // Desired playback speed, default 1.0
+  speed?: number;
+
+  // Theme color
+  themeColor?: string;
+
+  // Play on hover
+  hover?: boolean;
+
+  // Time to wait between playback loops
+  intermission?: number;
+}
+
+export interface Manifest {
+  // Default animation to play
+  activeAnimationId?: string;
+
+  // List of animations
+  animations: ManifestAnimation[];
+
+  // Name of the author
+  author?: string | undefined;
```

Comment:
`undefined` type is not needed, as `?` optional operator would infer the type to `string | undefined`